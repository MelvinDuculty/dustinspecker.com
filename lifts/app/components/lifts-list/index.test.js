import LiftsList from '.'
import React from 'react'
import renderer from 'react-test-renderer'

test('LiftsList', () => {
  const props = {
    lifts: [
      {
        id: '73',
        name: 'squats',
        workWeight: 300
      }
    ]
  }

  const component = renderer.create(
    <LiftsList {...props} />
  )

  const tree = component.toJSON()

  expect(tree).toMatchSnapshot()
})
