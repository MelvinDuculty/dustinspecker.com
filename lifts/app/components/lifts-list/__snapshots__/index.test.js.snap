// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LiftsList 1`] = `
<div
  className="MuiPaper-root-1 MuiPaper-elevation0-3 MuiPaper-rounded-2"
>
  <nav
    className="MuiList-root-28 MuiList-padding-29"
  >
    <div
      className="MuiPaper-root-1 MuiPaper-elevation1-4 MuiPaper-rounded-2"
    >
      <a
        className="MuiButtonBase-root-42 MuiListItem-root-32 MuiListItem-default-35 MuiListItem-gutters-39 MuiListItem-button-40"
        href="#/73"
        onBlur={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        role="button"
        tabIndex="0"
      >
        <div
          className="MuiListItemText-root-45 MuiListItemText-inset-46"
        >
          <h3
            className="MuiTypography-root-51 MuiTypography-subheading-58 MuiListItemText-primary-48"
          >
            Squats
          </h3>
          <p
            className="MuiTypography-root-51 MuiTypography-body1-60 MuiTypography-colorTextSecondary-73 MuiListItemText-secondary-49"
          >
            300 lbs
          </p>
        </div>
        <span
          className="MuiTouchRipple-root-75"
        />
      </a>
    </div>
  </nav>
</div>
`;
